docs/use-cases-monitoring.md

```diff
@@ -0,0 +1,43 @@
+---
+id: use-cases-monitoring
+title: Monitoring and Periodic Execution
+sidebar_label: Monitoring
+---
+
+
+## Motivation
+
+There is often a need for monitoring and periodic maintenance of IT systems on top of [infrastructure provisioning](./use-cases-provisioning). **Polling** is executing a regular action to check for a state change, for example:
+
+- Pinging a host to make sure it's online and responsive.
+- Once-per-minute health checks of a production deployment.
+- Polling an API for a specific resource to become available.
+- Triggering and executing periodic backups.
+- Pushing configuration updates when they become available.
+- Failing over in an active-passive setup when the primary instance becomes unhealthy.
+
+Monitoring is often an example of periodic execution of business logic, also known as a **distributed cron** engine.
+
+## Benefits of Temporal
+
+Temporal provides guaranteed execution, sophisticated error handling, flexible retry policies, and visibility into execution history for periodic workflow executions.
```

Comment:
I think this is missing the practical side of things. I think a code example wouldn't hurt for this one (either manually implemented CRON using Temporal.sleep or the official cron mechanism). Not super strongly opinioned here.

Reply:
Expanded a little bit... Let's track code examples as a follow-up.